.github/workflows/maven.yml

```diff
@@ -0,0 +1,13 @@
+name: Verify
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    name: Verify
+    uses: jetty/.github/.github/workflows/maven-ci.yml@main
+    with:
+      jdk-matrix: '[ "11", "17', "21" ]'
```

Comment:
JDK 11 only here.

Reply:
why? it's used with jetty 12 so should be tested with 17 and 21 which are used by jetty 12.